tests/BaseTestCase.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommentSoldTest;
```

Comment:
This should be `Tests` as configured by `composer.json`. I _think_ that should be all you need to autoload the test file.. if not, I can help you debug it, if you want.